h2/src/main/org/h2/util/SourceCompiler.java

```diff
@@ -189,6 +226,24 @@ public Method getMethod(String className) throws ClassNotFoundException {
         return null;
     }
 
+    public Object invoke(String className, final Object... args) throws Exception {
```

Comment:
rather call this getTrigger, since it is specific to that purpose now, and drops the args argument, since we don't need it yet

Reply:
The class is called SourceCompiler in the package util, IMHO it shouldn't reference Trigger which is higher level.